events/meetings/2025-09/program.md

```diff
@@ -16,36 +16,72 @@ meeting_nr: 202509
 
 ## Schedule
 
-The schedule is not yet ready.
+The schedule will be adjusted as we go along during the week. The idea is
+to be flexible, and adjust to your actual needs and interests as we go.
+Exercises can also be used to work on your personal OSCAR specific
+interests with support by the organizers.
 
-<!--
+<style>
+ul {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+</style>
 
-The schedule will be adjusted as we go along, so please watch out for changes.
+### Monday, September 15
+- 12:00 Lunch
+- 13:30 Getting started with Julia/Oscar/Editors etc.
+- 14:00 Guided session: **First steps with OSCAR** (with a focus on mathematical problems)
+- 15:30 Coffee break
+- 16:00 Exercises
+- 18:00 Dinner
+- 19:30 ⚡ Lightning️ Round of Introductions
+   - *be ready to state your name, affiliation, math interests
+       and why you need OSCAR in under 30 seconds 😁*
 
-### Monday, September 6
-  - 09:00 Time to install stuff, try Gather.town, socialize, ...
-  - 12:30 Lunch break
-  - 16:30 Evening standup
+### Tuesday, September 16
+- 08:00 Breakfast
+- 09:00 Guided session: **Exploring more of OSCAR's functionality (structure/subsystems)**
```

Comment:
I think this sounds a  bit dry and technical right now. Not quite the "Here we should you which wonderful mathematics OSCAR can do".... perhaps someone has an idea for a better title?

Reply:
"Oscar at your service -- advanced interactive functionality"